### DISCUSSION TITLE
Add swagger support

### DISCUSSION THREAD

**Discussion by maor-da** [stephenberry/glaze#411] [Ideas]
It seems Glaze is ideal for swagger codegen support for the CPP version. 
My suggestion is to create generator support which takes the lead in the area of json parsers, right now the only support is
for CPPREST

**Comment by maor-da** [comment]
And there is this which implements `nlohmann`
https://app.quicktype.io/

**Reply by stephenberry** [reply]
@maor-da,

Could you provide a small example of what you're looking for? I've looked at swagger before, but never used the specification. And, are you asking about C++ codegen to match the API? This would be interesting.

I don't have much time to devote to this right now, but I am interested in considering it further.

Thanks!
